tests/extract/s3_extract/test_stage_import.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 from mock import call
-from extract.s3_extract.stage_import import extract_from_stage, diagnostics_import, get_diagnostics_pattern
+from extract.s3_extract.stage_import import extract_from_stage, diagnostics_import, get_diagnostics_pattern, licenses_import
```

Comment:
@ifoukarakis , could we remove `diagnostics_import`?

Reply:
Can you please specify what you mean with remove? If you're talking about the DAG, it is currently used, so most likely is needed. If you're talking about separating these tests to two files, let's do any refactoring in a separate PR.